src/sensors.cpp

```diff
@@ -642,4 +675,12 @@ bool Sensors::OutlierFilter::update(float new_val, float *val)
   }
 }
 
+void Sensors::update_battery_monitor_multipliers()
+{
+  float voltage_multiplier = this->rf_.params_.get_param_float(PARAM_BATTERY_VOLTAGE_MULTIPLIER);
+  float current_multiplier = this->rf_.params_.get_param_float(PARAM_BATTERY_CURRENT_MULTIPLIER);
+  this->rf_.board_.battery_voltage_set_multiplier(voltage_multiplier);
+  this->rf_.board_.battery_current_set_multiplier(current_multiplier);
```

Comment:
Again, would it be better to just store this multiplier in the sensors module (or just get the parameter value every time), rather than pass it all the way down to airbourne driver?